folio-org/stripes-core
# STCOR-621: add configuration option for Help link

Adds the ability to specify a the URL used for the Help link in the main menu bar, by allowing an extra value, config.helpUrl, in stripes.config.js.  Without this value specified, falls back to the current default of 'https://docs.folio.org/docs'

To Test:
1. without editing stripes.config.js, deploy.  The link in the main nav bar should remain unchanged and active.
2. edit stripes.config.js in include helpUrl in the config section.  Point to the URL of your choice
3. deploy, and test.  You should now see the Help icon linking to your URL
